python/sanlock.c

```diff
@@ -320,8 +320,9 @@ static PyObject *
 py_init_lockspace(PyObject *self __unused, PyObject *args, PyObject *keywds)
 {
     int rv, max_hosts = 0, num_hosts = 0, use_aio = 1;
-    const char *lockspace, *path;
+    const char *path;
     struct sanlk_lockspace ls;
+    PyObject *lockspace;
```

Comment:
Changing the code should come after we update the tests. Doing changes before tests lead to broken tests that pass when the actual code fail.

Reply:
code and test changed in same patch (squashed test here)